packages/design-system-tokens/package.json

```diff
@@ -6,7 +6,6 @@
   "devDependencies": {
     "@types/node": "^17.0.22",
     "ts-node": "^10.4.0",
-    "tslib": "^2.3.1",
```

Comment:
I wasn't sure about this one, but I think you're right that we don't need it anymore. Looks like we don't have [this importHelpers config option](https://github.com/microsoft/tslib?tab=readme-ov-file#usage) anymore, though at one point we might have. It's not like a normal dependency where you can just check for import references to it, and even if we remove it, it's still installed indirectly through a boatload of other dependencies.